### PR TITLE
improve compose setup

### PR DESCRIPTION
**Pull request by vordimous**
## Purpose
update IS and compose version, use local mail,
remove the need for the dockerfile build with volumed jar get
update readme instructions

### CODE REVIEW

**Comment by chirangaalwis** [comment]
@vordimous let's change the target branch of the PR to `5.11.x` development branch.

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    depends_on:
+      - boostrap
+      - mysql
+      - mailserver
+
+volumes:
+  # Downloaded mysql jar
+  mysql-connector-jar:
+  # DB persisted data
+  mysql-datavolume:
```
Comment (chirangaalwis) [inline docker-compose/docker-compose.yml:63]: @vordimous please add a missing EOF line.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: docker-compose/docker-compose.yml (hunk 1 above)
Comment (vordimous) [reply docker-compose/docker-compose.yml]: fixed

File: docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  # Local Mailserver
+  mailserver:
+    image: mailhog/mailhog
+    ports:
+      - "8025:8025"
+      - "1025:1025"
+
+  # Identity Server
+  identity-server:
+    image: docker.wso2.com/wso2is:5.11.0.0 #5.11.0.0 is the latest 5.11 version alias
```
Comment (jyoakum) [inline docker-compose/docker-compose.yml:42]: Is 5.11.0.0 an alias to the latest stable version or is it an alias to a development version?

Comment (vordimous) [reply docker-compose/docker-compose.yml:42]: Per @chirangaalwis it is the latest stable version.

**Pull request closed without merging**